lib/Net/GitHub/V3/Repos.pm

```diff
@@ -197,6 +197,10 @@ my %__methods = (
     # http://developer.github.com/v3/repos/statuses/
     list_statuses => { url => "/repos/%s/%s/statuses/%s" },
     create_status => { url => "/repos/%s/%s/statuses/%s", method => 'POST', args => 1 },
+
+    # https://developer.github.com/v3/repos/deployments
+    create_deploy => { url => "/repos/%s/%s/deployments/", method => 'POST', args => 1 },
+    update_deploy => { url => "/repos/%s/%s/deployments/%s/statuses", method => 'POST', args => 1 },
```

Comment:
that's not very correct.

there should be 4 methods. list_deployments, create_deployment, list_deployment_statuses, create_deplayment_status

two GET two POST